pkg/webui/console/store/middleware/logics/packet-broker.js

```diff
@@ -0,0 +1,266 @@
+  process: async ({ action }, dispatch) => {
+    const {
+      payload: { id },
+      meta: { options: { fetchPolicies } = {} },
+    } = action
+    const ids = extractPacketBrokerIdsFromCombinedId(id)
+
+    const network = await fetchThroughPagination(
+      api.packetBroker.listNetworks,
+      ids.tenant_id ? { tenant_id_contains: ids.tenant_id } : undefined,
+      (result, acc) => result.networks.find(n => combinePacketBrokerIds(n.id) === id),
+      (result, acc) => Boolean(acc),
+    )
+
+    if (network && fetchPolicies) {
+      const fetchHomeNetworkRoutingPolicy = async () => {
+        try {
+          return await api.packetBroker.getHomeNetworkRoutingPolicy(ids.net_id, ids.tenant_id)
+        } catch (error) {
+          if (isNotFoundError(error)) {
+            return { home_network_id: ids }
+          }
+          throw error
+        }
+      }
+      const [forwarder, homeNetwork] = await Promise.all([
+        fetchThroughPagination(
+          api.packetBroker.listForwarderRoutingPolicies,
+          undefined,
+          (result, acc = []) => [...acc, ...result.policies],
+        ),
+        fetchHomeNetworkRoutingPolicy(),
+      ])
+      dispatch(packetBroker.getForwarderRoutingPoliciesSuccess(forwarder))
+      dispatch(packetBroker.getHomeNetworkRoutingPolicySuccess(homeNetwork))
+    }
+
+    return network
+  },
+})
+
+const getPacketBrokerNetworksLogic = createRequestLogic({
+  type: packetBroker.GET_PACKET_BROKER_NETWORKS_LIST,
+  latest: true,
+  process: async ({ action }, dispatch) => {
+    const {
+      payload: {
+        params: { page, limit, query, withRoutingPolicy },
+      },
+      meta: {
+        options: { fetchPolicies = {} },
+      },
+    } = action
+
+    const data = await api.packetBroker.listNetworks({
+      page,
+      limit,
+      name_contains: query,
+      tenant_id_contains: query,
+      with_routing_policy: withRoutingPolicy ? 'true' : 'false',
+    })
+
+    if (fetchPolicies) {
+      const [defaultPolicy, forwarder, homeNetwork] = await Promise.all([
+        getDefaultRoutingPolicy(),
+        fetchThroughPagination(
+          api.packetBroker.listForwarderRoutingPolicies,
+          undefined,
+          (result, acc = []) => [...acc, ...result.policies],
+        ),
+        fetchThroughPagination(
+          api.packetBroker.listHomeNetworkRoutingPolicies,
+          undefined,
+          (result, acc = []) => [...acc, ...result.policies],
+        ),
+      ])
+      dispatch(packetBroker.getHomeNetworkDefaultRoutingPolicySuccess(defaultPolicy))
+      dispatch(packetBroker.getForwarderRoutingPoliciesSuccess(forwarder))
+      dispatch(packetBroker.getHomeNetworkRoutingPoliciesSuccess(homeNetwork))
+    }
+
+    return { entities: data.networks, totalCount: data.totalCount }
+  },
+})
+
+const getPacketBrokerForwarderPoliciesLogic = createRequestLogic({
+  type: packetBroker.GET_FORWARDER_ROUTING_POLICIES,
+  process: async ({ action }) => {
```

Comment:
```suggestion
  process: async () => {
```

Please remove unused parameters in this file